src/components/purpose-card.js

```diff
@@ -75,10 +77,26 @@ class PurposeCard extends React.Component {
     })
   }
 
+  handleEditorChange = (content) => {
+    const { activeGroup, field } = this.props
+    const { data } = activeGroup
+    this.setState({[field]: content.level.content})
+  }
+
   render () {
     const { error, message, success } = this.state
     const { activeGroup, heading, field } = this.props
     const { data, inEditMode, hasEditRights } = activeGroup
+    const dict={
```

Comment:
Name the variables properly.
You can call it purposeField = {}
Add a space after and before `=`